smart-contract/assembly/contracts/internals/chunks.ts

```diff
@@ -101,3 +101,20 @@ export function _removeFile(
   _removeChunksRange(filePathHash, 0, newTotalChunks - 1);
   Storage.del(_getTotalChunkKey(filePathHash));
 }
+
+/**
+ * Deletes a chunks of a given file from storage.
+ * @param filePathHash - The hash of the file path.
+ * @throws If the chunk is not found in storage.
+ */
+export function _deleteFile(filePathHash: StaticArray<u8>): void {
+  const chunkNumber = _getTotalChunk(filePathHash);
+  for (let i: u32 = 0; i < chunkNumber; i++) {
+    assert(
+      Storage.has(_getChunkKey(filePathHash, i)),
+      'Chunk not found while deleting',
+    );
+    Storage.del(_getChunkKey(filePathHash, i));
+  }
+  _setTotalChunk(filePathHash, 0);
```

Comment:
we could completlly remove the total chunk key/value for this file

Reply:
yeah i was unsure, maybe even in the future we call a function that removes total chunk key, meta data etc etc